ckEditor/src/ckeditor.ts

```diff
@@ -156,7 +157,8 @@ const collaborativeEditorPlugins = [
 	TrackChanges,
 	TrackChangesData,
 	PresenceList,
-	DialogueCommentBox
+	DialogueCommentBox,
+	LLMFeedback
```

Comment:
I'm optimistic that this doesn't require updating ckeditor bundle versions, because the entire plugin is just adding a new command, but we'll need to keep an eye on it when we merge/deploy